packaging/darwin/amazon-cloudwatch-agent-ctl

```diff
@@ -192,7 +175,7 @@ cwa_config() {
      if [ "${config_location}" = "${ALL_CONFIG}" ]; then
           rm -rf "${JSON_DIR}"/*
      else
-          runDownloaderCommand="${CMDDIR}/config-downloader --output-dir ${JSON_DIR} --download-source ${config_location} --mode ${param_mode} --config ${COMMON_CONIG} --multi-config ${multi_config}"
+          runDownloaderCommand="${CMDDIR}/config-downloader --output-dir ${JSON_DIR} --download-source ${config_location} --mode ${mode} --config ${COMMON_CONIG} --multi-config ${multi_config}"
```

Comment:
Where in our code do we validate the mode parameter? What happens if the user passes in an invalid mode? Do we emit any log message anywhere for that? Since this is in the control script, do we want to gracefully recover and continue or fail fast if the mode is invalid?